modules/check_in/app/controllers/check_in/v2/appointments_controller.rb

```diff
@@ -19,6 +19,7 @@ def index
         # combine the appointments payload with facilities and clinics data
 
         merge_facilities(appointments[:data])
+        merge_clinic(appointments[:data])
```

Comment:
does it make sense to combine these 2 in one method? they are both iterating through the appointments and fetching facilities/clinics.

Reply:
Please find the updates in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)